### PR TITLE
Prison headset

### PR DESCRIPTION
**Pull request by Icarus-The-Sun**
Adds basic headsets without encryption keys to the prisoner loadouts, mainly aimed to counteract times when security is unaware prisoners are present, or during times the prison is destroyed and prisoners cannot call for help. Works with both plasmamen and default loadouts.

EDIT: Modularizes the outfits to be under their own file.

## Changelog

:cl:
add: Adds headsets to the roundstart prisoner outfit.
/:cl:

### CODE REVIEW

**Comment by NullDagaf** [comment]
Modularize it. Otherwise, 👍 from me.

**Comment by mcmeiler** [comment]
Same as above. I have no strong opinions on prisoners being able to talk to the rest of the crew. Could be interesting.
~~Might even see a lynchmob if they're being annoying or using it to host a radio station or something.~~

**Comment by Icarus-The-Sun** [comment]
This _should_ modularize it, sorry that it took a bit, I'm unfamiliar with modularizing code!

File: code/modules/jobs/job_types/prisoner.dm
Comment (KathrinBailey) [inline code/modules/jobs/job_types/prisoner.dm:1]: Revert these changes, you are overriding them in the modular code thus you have no reason to remove them.

**Review by KathrinBailey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**